lib/Github/Api/Issue/Comments.php

```diff
@@ -11,28 +11,42 @@
  */
 class Comments extends AbstractApi
 {
+    /**
+     * Configure the body type.
+     *
+     * @param string $bodyType
+     */
     public function configure($bodyType = null)
     {
         switch ($bodyType) {
             case 'raw':
-                $header = sprintf('Accept: application/vnd.github.%s.raw+json', $this->client->getOption('api_version'));
+                $header = 'Accept: application/vnd.github.%s.raw+json';;
                 break;
 
             case 'text':
-                $header = sprintf('Accept: application/vnd.github.%s.text+json', $this->client->getOption('api_version'));
+                $header = 'Accept: application/vnd.github.%s.text+json';
```

Comment:
the variable does not store a header anymore. It will be a proper header only after the sprintf call.